### ISSUE TITLE
NullPointerException when specifying capabilityMatcher as a command line parameter for Grid hub

### ISSUE THREAD

**Issue by JohnA2** [SeleniumHQ/selenium#2851]
## Meta -
OS:  Windows 7

Selenium Version:  3.0.0-beta4

Browser:  N/A

Browser Version:  N/A

## Expected Behavior -
The hub should be able to use a custom CapabilityMatcher class specified as a command line parameter.
## Actual Behavior -
NullPointerException even when passing the standard org.openqa.grid.internal.utils.DefaultCapabilityMatcher class
## Steps to reproduce -
1. Start the hub using `java -jar selenium-server-standalone-3.0.0-beta4.jar -role hub -capabilityMatcher org.openqa.grid.internal.utils.DefaultCapabilityMatcher`
2. Get an exception

```
Exception in thread "main" java.lang.NullPointerException
        at com.beust.jcommander.JCommander.convertValue(JCommander.java:1305)
        at com.beust.jcommander.JCommander.convertValue(JCommander.java:1228)
        at com.beust.jcommander.ParameterDescription.addValue(ParameterDescription.java:241)
        at com.beust.jcommander.ParameterDescription.addValue(ParameterDescription.java:209)
        at com.beust.jcommander.JCommander.processFixedArity(JCommander.java:875)
        at com.beust.jcommander.JCommander.processFixedArity(JCommander.java:857)
        at com.beust.jcommander.JCommander.parseValues(JCommander.java:725)
        at com.beust.jcommander.JCommander.parse(JCommander.java:282)
        at com.beust.jcommander.JCommander.parse(JCommander.java:265)
        at com.beust.jcommander.JCommander.<init>(JCommander.java:210)
        at org.openqa.grid.selenium.GridLauncherV3$2.setConfiguration(GridLauncherV3.java:228)
        at org.openqa.grid.selenium.GridLauncherV3.buildLauncher(GridLauncherV3.java:142)
        at org.openqa.grid.selenium.GridLauncherV3.main(GridLauncherV3.java:68)
```

**Comment by mach6** [comment]
thanks for the finding. I'll take a look

**Comment by mach6** [comment]
Confirmed issue.
Note: This is also an issue in versions beta1-beta3

**Comment by JohnA2** [comment]
Thanks for a quick fix. I think the same issue exists for "-prioritizer" and the PrioritizerString class. Just making the class `public static` instead of `private` fixes the problem.

**Comment by mach6** [comment]
thanks @lukeis for picking this up

**Comment by lukeis** [comment]
@JohnA2 ah, yes. thanks. fixed with f83d4f24c9a276eaf0cde94fdf1819cfb80a6bd0

I think i'd rather keep with the pattern of putting converters in their own class / package than keeping the inner classes around.

**Comment by JohnA2** [comment]
Thanks again, @lukeis. Yes, if other converters are already in `org.openqa.grid.internal.utils.configuration.converters`, it makes more sense to also mo it there.
